platform/core/src/services/DisplaySetService/DisplaySetMessage.ts

```diff
@@ -0,0 +1,59 @@
+const CODES = {
+  NO_VALID_INSTANCES: 1,
+  NO_POSITION_INFORMATION: 2,
+  NOT_RECONSTRUCTABLE: 3,
+  MULTIFRAME_NO_PIXEL_MEASUREMENTS: 4,
+  MULTIFRAME_NO_ORIENTATION: 5,
+  MULTIFRAME_NO_POSITION_INFORMATION: 6,
+  MISSING_FRAMES: 7,
+  IRREGULAR_SPACING: 8,
+  INCONSISTENT_DIMENSIONS: 9,
+  INCONSISTENT_COMPONENTS: 10,
+  INCONSISTENT_ORIENTATIONS: 11,
+  INCONSISTENT_POSITION_INFORMATION: 12,
+};
+
+class DisplaySetMessage {
+  id: number;
+
+  constructor(id: number) {
+    this.id = id;
+  }
+
+  public static get CODES() {
+    return CODES;
+  }
+}
```

Comment:
I don't understand the purpose for this class, the list is enough

Reply:
The idea is in the future put more attributes in this message class